acinclude.m4

```diff
@@ -36,6 +36,21 @@ AC_DEFUN([MK_AM_DISABLE_INTEGRATION_TESTS], [
                    [], [CPPFLAGS="$CPPFLAGS -DENABLE_INTEGRATION_TESTS"])
 ])
 
+AC_DEFUN([MK_AM_CHECK_LIBC_FUNCS], [
+  AC_CHECK_FUNCS([ \
+    err \
+    errx \
+    warn \
+    warnx \
+    getopt \
+    getopt_long \
+    getopt_long_only \
+    gmtime_r \
+    strcasecmp \
+    strtonum \
+  ])
```

Comment:
For each of these functions, if the function is not present on the system, we supply our own.